docs/configuration.md

```diff
@@ -411,12 +389,15 @@ insert newlines.
 
 ### `danglingParentheses`
 
-While this parameter is not technically under the `newlines` section, it logically belongs there.
+While this parameter is not technically under the `newlines` section, it
```

Comment:
These formatting changes come from Prettier, I have "format on save" enabled in VS Code in the Scalafmt repo. 

Maybe we should enforce prettier on all `*.md` files  in CI 🤔

Reply:
how would we do that? :)

Reply:
I took a stab here https://github.com/scalameta/scalafmt/pull/1840